.github/workflows/release-desktop-winms.yml

```
@@ -10,16 +10,17 @@
         node-version: [20.x]
+        arch: [x64, arm64]
 
     if: ${{ !github.event.workflow_run || (github.event.workflow_run && github.event.workflow_run.conclusion == 'success') }}
     steps:
       - name: Show executed time
         run: |
-          echo "Executed at: $(date '+%Y-%m-%d %H:%M:%S')"
+          Write-Host "Executed at: $(Get-Date -Format 'yyyy-MM-dd HH:mm:ss')"
 
       - name: Checkout Source Code
         uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)